packages/connect/package.json

```diff
@@ -23,9 +23,9 @@
     "prebuild": "yarn clean && yarn downloadSpecs",
     "pretest": "yarn build",
     "test": "yarn test:unit",
-    "test:all": "node --experimental-vm-modules $(yarn bin)/jest --colors --silent --coverage dist/",
-    "test:unit": "node --experimental-vm-modules $(yarn bin)/jest --colors --silent --coverage dist/ --config=jest.config.unit.ts",
-    "test:integration": "node --experimental-vm-modules $(yarn bin)/jest --colors --silent --coverage dist/ --config=jest.config.integration.ts",
+    "test:all": "../../bin/node-esm $(yarn bin)/jest --colors --silent --coverage dist/",
```

Comment:
I would prefer to avoid accessing bin directly and use the `node` command as it was before. Also I would avoid `--no-warnings`. It is better to see warnings when they occur (also its not like we will be altering the flags of this too often)

Reply:
we had experimental warnings for using modules >10 times in the testoutput before this.  This only hides the node warnings not any warnings from packages

Reply:
I did not initially realized that this `bin` is ours.. I got confused that it dives into node_modules or something..

Reply:
Yup it's a tiny wrapper script so we can change the flags we use in one place and avoid having long unreadable scripts in the package json